models/volunteer/volunteer.py

```diff
@@ -63,6 +63,12 @@ class Volunteer(db.Model, UserMixin):
         lazy="dynamic",
     )
 
+    def __repr__(self):
+        return "<Volunteer {0}<{1>>".format(self.user.name, self.user.email)
```

Comment:
You've got a > instead of a } there.

(I'd probably use [f-strings](https://docs.python.org/3/tutorial/inputoutput.html) for this now - they're supported on our python version and I think they'd be a bit clearer)

Reply:
Updated to f-strings now